### PR TITLE
removed redundancy in makefile

### PR DESCRIPTION
**Pull request by jocardeMSFT**
Self explanatory, simple housekeeping fix

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -14,7 +14,7 @@ export GO111MODULE=on
 #
 PKG := 
 
-all: format test unittest
+all: format test
```
Comment (mduppre) [inline Makefile:17]: While you're at it, I don't think there is any need to have `unittest` at all, is there? Let's just remove `test: unittest` and rename the `unittest` target to `test`?

**Review by mduppre — DISMISSED** [review]

File: Makefile (hunk 1 above)
Comment (mduppre) [reply Makefile:17]: Sure, I generally don't think it's worth leaving placeholders for future changes to come (which often never come), but I don't feel strongly at all about this tiny instance :-)

**Review by blairurish — DISMISSED** [review]

**Review by blairurish — APPROVED** [review]

**Review by halleremash — APPROVED** [review]

**Pull request merged**